src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    if (newRobot[robot[key]]) {
+      return null;
+    } // I don't understand how this part works, I found solution
+    // in internet, but still can't explain how
+    // *if* works without condition
```

Comment:
remove comments from your code, if you have any questions - feel free to ask in the slack chat, actually i didn't get your question